worldedit-core/src/main/java/com/sk89q/worldedit/command/GeneralCommands.java

```diff
@@ -448,6 +448,31 @@ public void placement(Actor actor, LocalSession session,
         placementImpl(actor, session, new Placement(placementType, offset));
     }
 
+    @Command(
+        name = "/ignoreybounds",
+        aliases = {"/iyb"},
+        desc = "Toggles or sets whether selections can extend outside the world's Y boundaries"
```

Comment:
This description is kind of misleading. It implies that e.g. `//outset` would also be allowed to go beyond the Y bounds, but this is not true since we will clamp it in the region impl with `recalculate`.

Now, this could just be reworded to state it only affects `pos` commands, but I think if there's going to be support for out-of-bounds regions, it should be comprehensive, not a one-off thing we check only at the command level, relying on us incidentally not recalculating regions in certain places. The regions should be constructed with a flag to control if they clamp, so that flipping this makes _all_ selection interactions clamp or not.

Potentially this also implies that certain commands may need re-implementation to support working outside the worlds Y bounds.